lib/deployer.rb

```diff
@@ -24,53 +24,53 @@ def initialize(environment:, repos:, tag: nil, before_command: nil)
     )
     @tag = tag
     ensure_tag_present_in_all_repos! if tag
-    prompt_user_for_branch_confirmation!
-    prompt_user_for_approval_confirmation!
   end
 
   # rubocop:disable Metrics/CyclomaticComplexity
   def deploy_all
-    puts "Repositories: #{repos.map(&:name).join(', ')}"
+    render_markdown('***')
+    render_markdown("# Deploying the following repositories (#{tag || 'default branch'})")
+    render_markdown(repos.map { |repo| "* #{repo.name}" }.join("\n"))
+
+    prompt_user_for_branch_confirmation!
+    prompt_user_for_approval_confirmation!
```

Comment:
Move these beneath the deploy-related header so it's clear what they pertain to.